docs/md/docs/2020-11-02-turn-the-senior-around.md

```diff
@@ -0,0 +1,68 @@
+# Turn The Senior Around
+
+As engineering managers we take many considerations into account, two of the main ones are:
+- Delivering on time
+- Make our people grow
+
+During my experience as an engineering manager and as an engineer all the managers I have met saw those subjects critical and invested a lot in improving them.<br>
+Another consensus among them all was that senior engineers are a key enabler to both topics, they are the main players the team lead can rely on to achieve those goals.<br>
+I spoke a bit about the role of senior engineer in my [The Last Dance blog](/docs/2020-07-09-last-dance) and this blog relates to it - I want to discuss about __*how*__ should engineering managers utilize their senior engineers to achieve those goals.
```

Comment:
`short rephrase I spoke a bit about the senior engineer's role in my`

Reply:
I'd replace `utilize` as it may sound like senior engineers are a tool used by the engineering manager. how about the following: I want to discuss how can engineering manager successfully achieve teams' goals by harnessing the power of their senior engineers?